File: Projects/EscapeTheStudyroom/Useable_Thing.py
from Thing import Thing

class Useable_Thing(Thing):
    def __init__(self,arg1,arg2,arg3):
        super().__init__(arg1,arg2) # == self.description = arg1; self.ID = arg2
        self._use = arg3
    
    def use(self):
        return

File: Projects/JoshgasIntelliJ/src/Items/UseableThing.java
package Items;

public class UseableThing extends Thing {

    String _use_msg = "Nothing happened...";

    public UseableThing(String arg1, String arg2){ //arg1 = desription, arg2 = use_msg
        super._description = arg1;
        _use_msg = arg2;
    }

    public UseableThing(){

    }

    public String use(){
        return _use_msg;
    }
}
